app/src/main/java/org/oppia/android/app/player/state/StateFragmentPresenter.kt

```diff
@@ -152,6 +156,10 @@ class StateFragmentPresenter @Inject constructor(
     return binding.root
   }
 
+  fun handleOnSaveInstanceState(bundle: Bundle) {
+    recyclerViewAssembler.saveState(bundle)
+  }
```

Comment:
We should also do this for the question player, though you may need to add a TODO to add tests for those. I think there are a few questions with hints & solutions that might allow you to trigger the failure case, but you'll need to find a question seed that reliably opens with the question you'd want to test with.

Reply:
Added it SaveState and RestoreState for the question player.
I can add the QuestionPlayer testing too,  but we only have Local tests for it and this bug is related to configuration change.